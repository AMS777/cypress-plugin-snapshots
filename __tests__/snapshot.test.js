const { subjectToSnapshot } = require('../snapshot');

describe('snapshot', () => {
  describe('subjectToSnapshot', () => {
    it('normalizes', () => {
      const normalized = subjectToSnapshot({
        foo: 'bar',
        bar: 'foo',
        sub: {
          foo: 'bar',
          bar: 'foo',
          asubsub: {
            foo: 'bar',
            bar: 'foo',
          }
        }
      }, true);

      expect(normalized).toMatchSnapshot();
    });
  });
});