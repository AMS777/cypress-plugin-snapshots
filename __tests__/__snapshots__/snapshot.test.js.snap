// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot subjectToSnapshot normalizes 1`] = `
Object {
  "bar": "foo",
  "foo": "bar",
  "sub": Object {
    "asubsub": Object {
      "bar": "foo",
      "foo": "bar",
    },
    "bar": "foo",
    "foo": "bar",
  },
}
`;
